This file is part of MXE.
See index.html for further information.

From c68b55c600c04a42bdd31e869b1a28d90868c2aa Mon Sep 17 00:00:00 2001
From: Timothy Gu <timothygu99@gmail.com>
Date: Sat, 27 Sep 2014 15:46:52 -0700
Subject: [PATCH] Work-around qmake bundled with Qt 4 not supporting macro
 expansion

---
 src/serialport/serialport.pro | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/serialport/serialport.pro b/src/serialport/serialport.pro
index feb6e9c..6500ca7 100644
--- a/src/serialport/serialport.pro
+++ b/src/serialport/serialport.pro
@@ -14,6 +14,10 @@ greaterThan(QT_MAJOR_VERSION, 4) {
     include($$PWD/qt4support/install-helper.pri)
     CONFIG += module create_prl
     mac:QMAKE_FRAMEWORK_BUNDLE_NAME = $$TARGET
+    load(moc)
+    # qmake bundled with Qt 4 doesn't support macro expansion.
+    # Work around it here.
+    QMAKE_MOC = $$QMAKE_MOC -DQT_DEPRECATED_SINCE=1
 }
 
 PRECOMPILED_HEADER =
-- 
1.8.3.2

